.vscode/launch.json

```diff
@@ -71,22 +71,6 @@
                 "RADIUS_ENV": "self-hosted"
             }
         },
-        {
-            "name": "Launch Deployment Engine",
-            "type": "coreclr",
-            "request": "launch",
-            "preLaunchTask": "Build Deployment Engine",
-            "program": "${workspaceFolder}/../deployment-engine/src/DeploymentEngine/bin/Debug/net6.0/arm-de.dll",
-            "args": [],
-            "cwd": "${workspaceFolder}/../deployment-engine/src/DeploymentEngine",
-            "stopAtEntry": false,
-            "env": {
-                "ASPNETCORE_URLS": "http://localhost:5017",
-                "ASPNETCORE_ENVIRONMENT": "Development",
-                "KUBERNETESBICEPEXTENSIBILITYURL": "http://localhost:5017/api",
-                "RADIUSBACKENDURI": "http://localhost:9000",
-            },
-        },
```

Comment:
Can we leave this here? It's still useful. 

That gives users the choice of either:

- Using the DE from a cluster
- Using the DE from local source code